composer.json

```diff
@@ -9,6 +9,7 @@
     },
     "license": "GPL-2.0+",
     "require": {
+        "php": ">=7.1",
```

Comment:
We had a discussion in the team about moving to PHP 7.1 so we can use strict typing. Everyone was in favor, we are assuming that if people are developing a project that is using RDF data storage then they are most likely to have control over their servers, and can install a recent PHP version too.